spectrum/src/main/java/com/thebluealliance/spectrum/internal/ColorItem.java

```diff
@@ -163,7 +174,12 @@ private Drawable createBackgroundDrawable() {
         GradientDrawable mask = new GradientDrawable();
         mask.setShape(GradientDrawable.OVAL);
         if (mOutlineWidth != 0) {
-            mask.setStroke(mOutlineWidth, ColorUtil.isColorDark(mColor) ? Color.WHITE : Color.BLACK);
+            if (mOutlineColor != -1) {
+                mask.setStroke(mOutlineWidth, mOutlineColor);
+            }
+            else {
```

Comment:
Keep your `else` on the same line as the preceding curly brace to match our coding style :)